functions/sql-update/index.js

```diff
@@ -0,0 +1,74 @@
+'use strict'
+
+const process = require('process')
+const sqlService = require('../lib/sql/sql.service')
+const { DONT_DROP_REDIS, TYPES } = sqlService
+
+if (process.env.NODE_ENV !== 'production') {
+  require('dotenv').config()
+}
+
+const schema = `[mtc_admin]`

```

Comment:
minor one: To the best of my knowledge it is always going to be `mtc_admin` so that makes this variable declaration kinda redundant

Reply:
agreed.  i removed these from the sql service a while back

Reply:
Hmmmm... I’ve removed it, but it’s still declared separately in `admin/services/data-access/sql.service.js`:
```
sqlService.adminSchema = '[mtc_admin]'
```

...which is why I declared it separately. Checking the latest master, just in case this was a recent change, it is still there - and then used 255 times across all the queries. Perhaps it's in a branch that hasn't been merged yet...?

If `mtc_admin` is the only schema we ever intend to use, then it’s quite right to remove that declaration. However, if the expectation is that the removal has already been done, that does not appear to be the case - at least, not everywhere.

Reply:
i'm wondering if my changes were scrapped / overwritten then, because this was a while back.  I've always been of the opinion that it's overkill, as it won't be changed.